### PR TITLE
Handled misspelled words

### PR DESCRIPTION
**Pull request by dixyushi**

### CODE REVIEW

File: .spellcheck.yml [hunk 1]
```diff
@@ -35,5 +65,5 @@ matrix:
       context_visible_first: true
       delimiters:
       # Ignore multiline content between [source, *] and blank line
-      - open: '\[source,.*\]'
-        close: '^\s*$' 
+      - open: '\[source, .*\]\r?\n\s*----'
+        close: '[^-]*'
```
Comment (ssarmokadam) [inline .spellcheck.yml:69]: Update to latest regex 
```
 - pyspelling.filters.context:
      context_visible_first: true
      delimiters:
      # Ignore multiline content between [source, *] and blank line
      - open: '\[source,.*\]\r?\n\s*----'
        close: '\s*----'
```

File: documentation/architecture_guide.asciidoc [hunk 2]
```diff
@@ -141,7 +141,7 @@ The provided _Data Layer_ contains the abstract classes to inherit from. All new
 
 [[img-t-architecture]]
 .Architecture representation in deep
-image::images/laryer_arch_detail.png["technical architecture", width="750", link="images/laryer_arch_detail.png"]
+image::images/layer_arch_detail.png["technical architecture", width="750", link="images/layer_arch_detail.png"]
```
Comment (ssarmokadam) [inline documentation/architecture_guide.asciidoc:144]: is this link working now? is image getting loaded?

**Review by ssarmokadam — CHANGES_REQUESTED** [review]

File: documentation/architecture_guide.asciidoc (hunk 2 above)
Comment (dixyushi) [reply documentation/architecture_guide.asciidoc:144]: I have updated new image with new name. Now it's working.

File: .spellcheck.yml (hunk 1 above)
Comment (dixyushi) [reply .spellcheck.yml:69]: ok, I will

**Pull request merged**